app/views/layouts/partials/_header.html.haml

```diff
@@ -7,10 +7,22 @@
     %ul.menu
       %li
         = link_to 'Home', root_path
-      %li
-        = link_to 'Login', new_user_session_path
-      %li
-        = link_to 'Sign Up', new_user_registration_path
-      - if current_user && current_user.restaurants.any?
+      - if current_user
+        %li
+          %p Hello, #{current_user.email}
+        %li
+          = link_to 'Logout', destroy_user_session_path, method: :delete
+      - else
+        %li
+          = link_to 'Login', new_user_session_path
         %li
-          = link_to 'Dashboard', dashboard_path(current_user.restaurants.first)
+          = link_to 'Sign Up', new_user_registration_path
+
+
+          -# - elsif current_user && current_user.restaurants.any?
```

Comment:
That line was used to condition the dashboard link. Not sure if that's still the case.